README.md

```diff
@@ -29,12 +29,12 @@ slack.uploadFile({
 	title: 'README',
 	initialComment: 'my comment',
 	channels: 'XXXXX'
-}, function(err) {
+}, function(err, data) {
 	if (err) {
 		console.error(err);
 	}
 	else {
-		console.log('done');
+		console.log('Uploaded file details: ' + data);
```

Comment:
this will convert data into a string
My vote is
`console.log('Uploaded file details:', data);`
which will dump whatever data is, not just stringify it